app/mailers/contact_mailer.rb

```diff
@@ -1,8 +1,14 @@
 class ContactMailer < ApplicationMailer
-  default from: Rails.application.credentials.mailer_sender
+  default from: -> { contact_form_email }
```

Comment:
We should keep default from email.
We can not send email from other email addresses